cmd/promtool/main.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015 The Prometheus Authors
+// Copyright 2018 The Prometheus Authors
```

Comment:
We usually just leave years in existing copyright notices as they are (the full situation is a bit more complicated if you wanted to do it completely right, see: https://opensource.stackexchange.com/questions/2171/refreshing-shared-copyright-dates-in-apache-2-0-licensed-code)

Reply:
I think @brian-brazil asked me to update these on one of my PRs.